components/bounty/funding-section/fund-modal.tsx

```diff
@@ -36,19 +34,20 @@ export default function FundModal({
 
   const { processEvent } = useApi();
   const { handleFundBounty } = useBepro();
-  const {getDatabaseBounty, getChainBounty} = useBounty();
+  const { getDatabaseBounty } = useBounty();
   const { allowance, balance, decimals, setAddress, approve, updateAllowanceAndBalance } = useERC20();
 
-  const bountyId = state.currentBounty?.data?.contractId || state.currentBounty?.chainData?.id || "XX";
+  const bountyId = state.currentBounty?.data?.contractId || "XX";
```

Comment:
could this introduce weird errors if we fall on the "XX" case? (shouldn't we just remove it?) [i know it's old-code]